app/components/Settings/Settings.react.js

```diff
@@ -376,73 +375,43 @@ class Settings extends Component {
-                            )}
-                        </TabPanel>
-
-                        <TabPanel>
-                            {this.props.connectRequest.status === 200 ? (
-                                <div>
-                                    {httpsServerIsOK ? (
-                                        <div>
-                                            <div>
-                                                <p>
-                                                    {'No action required. Plotly has already generated a unique SSL certificate for this app.'}
-                                                </p>
-                                            </div>
-                                        </div>
-                                    ) : (
-                                        <div>
-                                            <p>
-                                                {`Plotly is automatically initializing a
-                                                  unique SSL certificate and URL for you.
-                                                  This can take up to 10 minutes.`}
-                                            </p>
-                                            <p>
-                                                {`It has been ${timeElapsed}`}.
-                                            </p>
-                                            <p>
-                                                {`Once this is complete, you'll be able to
-                                                  query your databases from `}
-                                                <Link href={`${plotlyUrl}/create?upload=sql`}>
-                                                        Plotly
-                                                </Link>
-                                                {'.'}
-                                            </p>
-                                        </div>
-                                    )}
+                                <div className="big-whitespace-tab">
+                                    <p>Please connect to a data store in the Connection tab first.</p>
                                 </div>
-                            ) : (
-                                <p>You must connect to a data store first.</p>
                             )}
                         </TabPanel>
 
                         <TabPanel>
                             {this.props.connectRequest.status === 200 ? (
-                                <div>
+                                <div className="big-whitespace-tab">
                                     {httpsServerIsOK ? (
-                                        <div id="test-ssl-initialized">
+                                        <div id="test-ssl-initialized" style={{textAlign: 'center'}}>
+                                            <img 
+                                                src="../../../images/architecture.png" 
```

Comment:
this is an odd URL. should / can this just be `/static/images/architecture.png`?